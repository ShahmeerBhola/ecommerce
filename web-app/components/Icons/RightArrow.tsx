import React from 'react';
import config from '../../config';

const RightArrow: React.FunctionComponent = () => {
  const red100 = config.colors['red-100'];
  return (
    <svg width="30" height="30" viewBox="0 0 50 50" fill="none" xmlns="http://www.w3.org/2000/svg">
      <rect x="1" y="1" width="48" height="48" stroke={red100} strokeWidth="2" />
      <path
        d="M30.0168 31L28.2218 29.2L31.201 26.2125L14 26.2125L14 23.6669L31.0807 23.6669L28.2218 20.8L30.0168 19L36 25L30.0168 31Z"
        fill={red100}
      />
    </svg>
  );
};

export default RightArrow;
