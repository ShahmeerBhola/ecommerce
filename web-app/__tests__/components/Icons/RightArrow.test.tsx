import React from 'react';
import renderer from 'react-test-renderer';
import RightArrow from '../../../components/Icons/RightArrow';

describe('RightArrow component', () => {
  test('it renders properly', () => {
    const tree = renderer.create(<RightArrow />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
