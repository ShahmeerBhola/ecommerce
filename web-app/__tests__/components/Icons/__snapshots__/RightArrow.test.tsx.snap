// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RightArrow component it renders properly 1`] = `
<svg
  fill="none"
  height="50"
  viewBox="0 0 50 50"
  width="50"
  xmlns="http://www.w3.org/2000/svg"
>
  <rect
    height="48"
    stroke="#F23939"
    strokeWidth="2"
    width="48"
    x="1"
    y="1"
  />
  <path
    d="M30.0168 31L28.2218 29.2L31.201 26.2125L14 26.2125L14 23.6669L31.0807 23.6669L28.2218 20.8L30.0168 19L36 25L30.0168 31Z"
    fill="#F23939"
  />
</svg>
`;
